pkg/storage/run_test.go

```diff
@@ -186,9 +184,8 @@ func TestRun_TypedParameterValues(t *testing.T) {
 		"foo":          5,
 	}
 
-	run.Parameters.Parameters = params
 	typed := run.TypedParameterValues()
-	require.Equal(t, len(params), len(typed))
+	require.Equal(t, len(run.Parameters.Parameters), len(typed))
```

Comment:
This is a future nit: `run.Parameters.Parameters` should have a different naming scheme, `run.ParametersSets.Parameters` or something more to what it really is.